target/linux/ramips/dts/E220.dts

```diff
@@ -0,0 +1,208 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "e220", "mediatek,mt7620a-soc";
+	model = "Maestro Wireless Solutions E220";
+
+    chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+        activity {
```

Comment:
wrong indention and spaces instead of tabs. Please check the whole file for such issues, the are a few more.